joelsantos-js.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Joel",
+  "url": "https://github.com/JoelSantos-JS",
+  "image": "https://github.com/account",
```

Comment:
Kindly fix the image URL to your image. Please use the below line of code instead of your URL.
"image": "https://avatars.githubusercontent.com/u/37488416?v=4"

Reply:
tanks